tests/ut-torchapi.cc

```diff
@@ -1453,8 +1508,6 @@ TEST(torchapi, service_train_csvts_nbeats_db)
   std::string uri = jd["body"]["predictions"][0]["uri"].GetString();
   ASSERT_EQ("../examples/all/sinus/predict/seq_2.csv #0_99", uri);
   ASSERT_TRUE(jd["body"]["predictions"][0]["series"].IsArray());
-  ASSERT_TRUE(jd["body"]["predictions"][0]["series"][0]["out"][0].GetDouble()
-              >= -2.0);
```

Comment:
Finally removed ? :)
Can't we just have at least a lower bound, e.g. -10 or -100? so if the value is NaN or -1e300 it still fails

Reply:
ok, done